CHANGELOG.txt

```diff
@@ -38,6 +38,7 @@ v27.0.00
         Fast Finder: added facilities as one of the searchable options with the fast finder.
         Messenger: Added search form and school year navigator to Manage Groups page for quicker group browsing.
         System: Replaced session call through $gibbon->session to $session. "$gibbon->session" is now deprecated.
+        Facilities: Added an 'active' column in the gibbonSpace table and changed pages to only show active Facilities.
```

Comment:
Here's a slightly less technical way to word this, for the end user who is also often less technical (our commit messages are a good place for more specific wording, since those are more dev-facing)
```suggestion
        Facilities: added the ability to toggle facilities between active and inactive, removing inactive facilities from general use
```